app/workers/rapid_ready_for_decision/form526_base_job.rb

```diff
@@ -135,10 +140,14 @@ def account(form526_submission)
     end
 
     def upload_pdf(form526_submission, pdf)
-      RapidReadyForDecision::FastTrackPdfUploadManager.new(form526_submission).handle_attachment(pdf.render)
+      RapidReadyForDecision::FastTrackPdfUploadManager
+        .new(form526_submission)
+        .handle_attachment(pdf.render, add_to_submission: release_pdf?(form526_submission))
     end
 
     def set_special_issue(form526_submission)
+      return if release_pdf?(form526_submission)
+
```

Comment:
The presumed logic for this is: if we don't want to release the pdf, then the claim shouldn't be fast-tracked (via the `RRD` special issue). Wondering if we should add a code comment to avoid anyone thinking this is a bug since generating the PDF and routing are arguably tangential (but related).

Reply:
Hm, it actually reads pretty straightforward to me, so I'm not inclined to add a comment unless you feel strongly.

I expect that we may eventually remove this mechanism anyway, unless in the asthma retro we decide that it's a pattern we want to keep and formalize for future contentions.

Reply:
Not strongly. Go ahead!